wemake_python_styleguide/visitors/ast/exceptions.py

```diff
@@ -26,6 +26,8 @@
 )
 from wemake_python_styleguide.visitors.base import BaseNodeVisitor
 
+TRIVIAL_EXCEPT_ARG_TYPES = (ast.Name, ast.Attribute)
```

Comment:
Let's make it a class-level contant like we always do. 
Example: https://github.com/wemake-services/wemake-python-styleguide/blob/3e679c1a23dc149b664657df5a6374af71fdd164/wemake_python_styleguide/visitors/ast/exceptions.py#L59

Reply:
Sorry I had to be more specific!